adapter/src/main/java/com/twilio/kudu/sql/KuduEnumerable.java

```diff
@@ -355,7 +354,7 @@ public void close() {
         scanStats.setTotalTimeMs();
         List<ScannerMetrics> scannerMetricsList = scanners.stream().map(scanner -> new ScannerMetrics(scanner))
             .collect(Collectors.toList());
-        scanStats.setScannerMetricsList(scannerMetricsList);
+        scanStats.addScannerMetricsList(scannerMetricsList);
```

Comment:
Were these two spots erasing scanner metrics and replacing them?

Reply:
Yes, queries that used KuduNestedJoin didn't had fewer scanner metrics than expected.